networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/BlobsSidecarsByRangeMessageHandler.java

```diff
@@ -228,7 +228,10 @@ SafeFuture<Optional<BlobsSidecar>> loadNextBlobsSidecar() {
               block ->
                   block
                       .map(SignedBeaconBlock::getRoot)
-                      .map(combinedChainDataClient::getBlobsSidecarByBlockRoot)
+                      .map(
+                          blockRoot ->
+                              combinedChainDataClient.getBlobsSidecarBySlotAndBlockRoot(
+                                  currentSlot.get(), blockRoot))
```

Comment:
could we modify 
```
boolean isComplete() {
      return currentSlot.get().equals(maxSlot)
```
to `greaterOrEquals`, please. it will add some safety